src/core/mac/mac_frame.cpp

```diff
@@ -1031,7 +1031,7 @@ void TxFrame::ProcessTransmitAesCcm(const ExtAddress &aExtAddress)
 
 exit:
     return;
-#endif // OPENTHREAD_MTD || OPENTHREAD_FTD
+#endif // OPENTHREAD_RADIO
```

Comment:
How about moving this to a different PR or commit? Seems very unrelated to rest :)